### ISSUE TITLE
EditorXR state of development

### ISSUE THREAD

**Issue by PixelDrennen** [Unity-Technologies/EditorXR#582]
#579 mentions that "this product is not supported" -- was wondering what this refers to. Is EditorXR still in development?

**Comment by Sandstedt** [comment]
Also curious if there is any initiative from Unity to bring this up again? Especially with all the hype around spatial computing now. I would love to be able to to use a stereoscopic display or VR for set dressing and level design.

**Comment by SeanBannister** [comment]
One of the unity staff announced on the forums that this was no longer in development. Shame as this looked great.

**Comment by haywirephoenix** [comment]
at this point I'd be more surprised if they actually finished something

**Comment by mtschoen-unity** [comment]
Indeed, we are no longer actively developing EditorXR. Some of the learnings and functionality from this project lives on in the XR Interaction Toolkit and VR/MR templates that we have shipped in the intervening years. Of course, for nostalgia's sake, you should still be able to load up Unity 2019.4 (and I think some earlier versions as well) to try out EditorXR. There was even a package manager release that you can install if you add `com.unity.editorxr` to your project by name in the Package Manager.

We appreciate your interest in EditorXR. While I'm also a little sad about how it turned out, it's still good to hear that users are interested in this functionality and the future of authoring in XR! There are some great standalone apps that have come out since we started working on EditorXR, like PolySketch. And again, I encourage you to explore the latest packages and templates for XR support in Unity. We've created some great simulation workflows to improve on iteration times while working with XR content. Of course, none of these things are a replacement of EditorXR, but we're still working to improve the state of the art in XR content creation.
